MainRepository/Program.cs

```diff
@@ -1,7 +1,68 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find the possible combination of queen in the n*n chess board
+class NQueen {
+   static int Count;
+   //Function to check if two queens attack each other or not
+   // Unicode of queen-\u2655
+   static bool IsSafe (char[,] board, int r, int c) {
+      // Checking the column 
+      for (int i = 0; i < r; i++) {
+         if (board[i, c] == '\u2655')
+            return false;
+      }
+      // Checking the left diagonal 
+      for (int i = r, j = c; i >= 0 && j >= 0; i--, j--) {
+         if (board[i, j] == '\u2655')
+            return false;
+      }
+      // Checking the right diagonal 
+      for (int i = r, j = c; i >= 0 && j < board.GetLength (0); i--, j++) {
+         if (board[i, j] == '\u2655')
+            return false;
+      }
+      return true;
+   }
+
+   // Printing the queen in the chess board
+   static void PrintSolution (char[,] board) {
+      Console.WriteLine ("┌───┬───┬───┬───┬───┬───┬───┬───┐");
+      for (int i = 0; i < board.GetLength (0); i++) {
+         Console.Write ("\u2502");
+         for (int j = 0; j < board.GetLength (1); j++)
+            Console.Write (board[i, j] == '\u2655' ? " \u2655 \u2502" : "   \u2502");
+         if (i == 7) {
+            break;
+         }
+         Console.WriteLine ();
+         Console.WriteLine ("├───┼───┼───┼───┼───┼───┼───┼───┤");
+      }
+      Console.WriteLine ();
+      Console.WriteLine ("└───┴───┴───┴───┴───┴───┴───┴───┘");
+   }
+
+   static void Nqueen (char[,] board, int r) {

```

Comment:
```suggestion
   static void NQueen (char[,] board, int r) {

```
You can change the class name to something else or this to PlaceQueen